DigitalLearningSolutions.Data/DataServices/UserDataService/DelegateUserDataService.cs

```diff
@@ -49,34 +53,10 @@ public partial class UserDataService
 
         public List<DelegateUser> GetDelegateUsersByUsername(string username)
         {
-            List<DelegateUser> users = connection.Query<DelegateUser>(
-                @"SELECT
-                        cd.CandidateID AS Id,
-                        cd.CandidateNumber,
-                        ct.CentreName,
-                        cd.CentreID,
-                        cd.DateRegistered,
-                        ct.Active AS CentreActive,
-                        cd.EmailAddress,
-                        cd.FirstName,
-                        cd.LastName,
-                        cd.Password,
-                        cd.Active,
-                        cd.Approved,
-                        cd.ProfileImage,
-                        cd.Answer1,
-                        cd.Answer2,
-                        cd.Answer3,
-                        cd.Answer4,
-                        cd.Answer5,
-                        cd.Answer6,
-                        cd.JobGroupID,
-                        jg.JobGroupName
-                    FROM Candidates AS cd
-                    INNER JOIN Centres AS ct ON ct.CentreID = cd.CentreID
-                    INNER JOIN JobGroups AS jg ON jg.JobGroupID = cd.JobGroupID
+            var users = connection.Query<DelegateUser>(
+                @$"{BaseSelectDelegateQuery}
                     WHERE cd.Active = 1 AND
-                         (cd.CandidateNumber = @username OR cd.EmailAddress = @username OR cd.AliasID = @username)",
+                         (cd.CandidateNumber = @username OR cd.EmailAddress = @username)",

```

Comment:
We should check this behaviour change in [887](https://softwiretech.atlassian.net/browse/HEEDLS-887)